.github/workflows/bot.yml

```diff
@@ -210,3 +210,62 @@ jobs:
         run: |
           HUDI_VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
           ./packaging/bundle-validation/ci_run.sh $HUDI_VERSION openjdk17
+
+  validate-release-candidate-bundles:
+    if: true
+    runs-on: ubuntu-latest
+    env:
+      HUDI_VERSION: 0.13.1-rc1
```

Comment:
by default can you set some invalid release(instead of 0.13.1 rc1), so that it will fail rather than succeeding by mistake.